lib/assessment/config.py

```diff
@@ -1,11 +1,11 @@
 VALID_LABELS = ["Extensive Evidence", "Convincing Evidence", "Limited Evidence", "No Evidence"]
 SUPPORTED_MODELS = ['gpt-4', 'gpt-4-0314', 'gpt-4-32k', 'gpt-4-32k-0314']
 LESSONS = {
-    "U3-2022-L10" : "1ROCbvHb3yWGVoQqzKAjwdaF0dSRPUjy_",
-    "U3-2022-L13" : "1kGHeY5LRpFJ9xVRoBEWbyOJyKm4wClqw",
+    # "U3-2022-L10" : "1ROCbvHb3yWGVoQqzKAjwdaF0dSRPUjy_",
+    # "U3-2022-L13" : "1kGHeY5LRpFJ9xVRoBEWbyOJyKm4wClqw",
     "U3-2022-L17" : "1WirJLIFgo-anxAz-kZXDVQ2Tl_8OuX22",
     "U3-2022-L20" : "115BHvZ1kJC2xhUSOBkLiE8DC1YgcjyRd",
-    "U3-2022-L23" : "12OJex4l9OhWrnbLenpvZAibtfiFWWdzx",
+    # "U3-2022-L23" : "12OJex4l9OhWrnbLenpvZAibtfiFWWdzx",
```

Comment:
did you mean to leave these commented out?

Reply:
I did, these files in google drive were not in a good state. This, along with your issues pulling down from google drive, are why I'd prefer to put the test files and actual results in s3 so we pull a more "final" version of the files with more explicit authentication than whatever gdown is doing

Reply:
this is fine, thank you for clarifying. it was an honest question :-) I think it is ok to switch to S3 in the next PR.